### PR TITLE
Set up architecture for access-control tests and automated first four test cases

### PR DESCRIPTION
**Pull request by theemj**

### CODE REVIEW

File: tests/accesscontrol/access_control_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"testing"
+
+	"github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/helper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/parameters"
+	_ "github.com/test-network-function/cnfcert-tests-verification/tests/accesscontrol/tests"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+func TestAffiliatedCertification(t *testing.T) {
```
Comment (shaior) [inline tests/accesscontrol/access_control_suite_test.go:19]: TestAffiliatedCertification --> TestAccessControl

File: tests/accesscontrol/helper/helper.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+			clientSet,
+			namespace,
+			timeout,
+		)
+		if err != nil {
+			failedNs = append(failedNs, namespace)
+		}
+	}
+
+	if len(failedNs) > 0 {
```
Comment (shaior) [inline tests/accesscontrol/helper/helper.go:26]: I think it will be better to see the error, maybe add err to the fmt.error?

**Review by shaior — COMMENTED** [review]
Looks good so far, just minor comments :+1:

**Review by sebrandon1 — APPROVED** [review]

File: tests/accesscontrol/access_control_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	_, reporterConfig := GinkgoConfiguration()
+	reporterConfig.JUnitReport = globalhelper.Configuration.GetReportPath(currentFile)
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CNFCert access-control tests", reporterConfig)
+}
+
+var _ = BeforeSuite(func() {
+	By("Create namespace")
+	err := namespaces.Create(parameters.TestAccessControlNameSpace, globalhelper.APIClient)
```
[the remark this answers is not in the document]
Comment (theemj) [reply tests/accesscontrol/access_control_suite_test.go:32]: The TestAccessControlNameSpace will be used for many access-control tests, while the ones defined in  access_control_namespace.go will only be used for those namespace-specific test cases. I can add a comment explaining this when I define the additional namespaces in access_control_namespace.go

[the remark this answers is not in the document]
Comment (shaior) [reply tests/accesscontrol/access_control_suite_test.go:32]: ok, I understand, then it looks good to me.

**Pull request merged**